tests/test_asset.py

```diff
@@ -80,6 +84,18 @@ def test_asset_update_metadata(asset_mock):
     assert updated_info["tags"] == ["othertag1", "othertag2"]
 
 
+def test_asset_update_metadata_should_return_same_with_no_values(asset_mock):
```

Comment:
is it not similar to `test_asset_update_metadata`?

Reply:
nop, one tests if the data is updated, the other test the other case (that was the source of the error), if not provided then no update return the same.

Reply:
You are right, I stupidly did not remove the inputs that were what I wanted to test. Thanks.